install_config/persistent_storage/persistent_storage_azure_file.adoc

```diff
@@ -51,25 +51,51 @@ $ mount
 If the `dir_mode` and `file_mode` permissions are set to `0755`,  change the
 default value `0755` to `0777` or `0775`. This manual step is required because
 the default `dir_mode` and `file_mode` permissions changed from `0777` to `0755`
-in {product-title} 3.9. The following examples show configuration files with the 
+in {product-title} 3.9. The following examples show configuration files with the
 changed values.
 
+.Considerations when using MySQL and PostgresSQL with Azure file
+* The owner UID of the Azure File mounted directory is different from the UID of a container.
+* Both MySQL and PostgreSQL containers change the file owner permission in the mounted directory. Because of the mismatch between the owner UID and container process UID, this operation fails. Therefore to run MySQL with Azure File:
+** Specify the UID in `runAsUser` variable in the pod specification file.
++
+[source, yaml]
+----
+spec:
+  containers:
+    ...
+  securityContext:
+    runAsUser: 1000125000
+----
+** Specify the `uid` in mount options:
++
+[source, yaml]
+----
+mountOptions:
+  - dir_mode=0700
+  - file_mode=0600
+  - uid=<conatiner_process_uid>
+  - gid=0
+----
+
+* Azure File does not support link:https://docs.microsoft.com/en-us/rest/api/storageservices/features-not-supported-by-the-azure-file-service[symbolic link]. Therefore when PostgreSQL creates a symbolic link in the Azure File directory, the pod fails to start.
```

Comment:
This makes it sound like it will happen and there's nothing you can do about it. Should it be "If PostgreSQL creates a symbolic link, the pod will fail"? Is there a way around this?